flight/PiOS/Common/pios_rcvr.c

```diff
@@ -143,12 +143,12 @@ void PIOS_RCVR_Active() {
       rcvr_last_wake = PIOS_DELAY_GetRaw();
       PIOS_Semaphore_Give(rcvr_activity);
     }
+  }
 #ifdef FLIGHT_POSIX
-    if (PIOS_Thread_FakeClock_IsActive()) {
-      PIOS_Thread_FakeClock_UpdateBarrier(100);
-    }
-#endif
+  if (PIOS_Thread_FakeClock_IsActive()) {
```

Comment:
Changed: any receiver update is OK to unblock, not just ones that happen a far enough wallclock time away.